pages/common/nginx.md

```diff
@@ -8,4 +8,4 @@
 
 - Start server with custom config and custom relative path base directory
 
-`nginx -c {{path to config}} -p {{relative path base dir}}`
+`nginx -c {{config_file}} -p {{relative/path/to/base/dir}}`
```

Comment:
So you mean it should be

`nginx -c {{config_file}} -p {{relative/path/base/dir}}`

instead of

`nginx -c {{config_file}} -p {{relative/path/to/base/dir}}`

Right?